root/etc/s6-overlay/s6-rc.d/init-wireguard-confs/run

```diff
@@ -39,7 +39,7 @@ DUDE"
         PEER_ID="peer_${i}"
       fi
       mkdir -p "/config/${PEER_ID}"
-      if [[ ! -f "/config/${PEER_ID}/privatekey-${PEER_ID}" ]]; then
+      if [[ ! -f "/config/${PEER_ID}/publickey-${PEER_ID}" ]]; then
```

Comment:
Hey @aptalca, neat solution!

I have two questions though:

1. Since removing private keys alone will not regenerate peer confs, additionally we'll need to edit peer confs manually to remove each private key, right?
2. What happens if peer/server confs regeneration is triggered (by changing any of the optional variables)? Won't the code fail if `privatekey-${PEER_ID}` doesn't exist?

Regarding `2.`, I thought of this simple change:
```suggestion
      if [[ ! -f "/config/${PEER_ID}/privatekey-${PEER_ID}" ]]; then
        echo "<insert_your_private_key_here>" > "/config/${PEER_ID}/privatekey-${PEER_ID}"
      fi
      if [[ ! -f "/config/${PEER_ID}/publickey-${PEER_ID}" ]]; then
```

Regarding `1.`, peer/server confs regeneration could be triggered/flagged inside the first "if" of the suggestion above.

Thanks a lot for the PR!